src/hearings/containers/request-hearing/date-priority-hearing/date-priority-hearing.component.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, Input, NO_ERRORS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { RefDataModel } from 'api/hearings/models/refData.model';
+import { Observable } from 'rxjs';
+import { ErrorMessage } from 'src/app/models';
+import { DatePriorityHearingComponent } from './date-priority-hearing.component';
+
+
+@Component({
+  selector: 'exui-hearing-parties-title',
+  template: '',
+})
+class MockHearingPartiesComponent {
+  @Input() public error: ErrorMessage;
+}
+
+describe('DatePriorityHearingComponent', () => {
+  let component: DatePriorityHearingComponent;
+  let fixture: ComponentFixture<DatePriorityHearingComponent>;
+  let router: Router;
+  const PRIORITIES: RefDataModel[] = [
+    {
+      key: 'string',
```

Comment:
confusing mock data. mock data contents should example what info should be there not the type